symbulate/distributions.py

```diff
@@ -533,16 +528,16 @@ def __init__(self, a, b, scale=None):
         self.xlim = (0, 1) # Beta distributions are not defined for x < 0 and x > 1
 
     def draw(self):
-        """A function that takes no arguments and 
+        """A function that takes no arguments and
             returns a single draw from the Beta distribution."""
 
-        return np.random.beta(self.a, self.b)
+        return (self.xmin + (self.xmax - self.xmin)*np.random.beta(self.a, self.b))
```

Comment:
Remove outer parentheses, make sure you have spaces surrounding all operations, like `*`.